packages/preview-middleware-client/src/adp/controllers/BaseDialog.controller.ts

```diff
@@ -181,6 +186,25 @@ export default abstract class BaseDialog<T extends BaseDialogModel = BaseDialogM
             );
             return;
         }
+        // 'changes.fragments' is the current folder structure where fragment changes are written. 
+        // following value is subjected to change if the folder structure changes
+        const template = `${this.rta.getFlexSettings()?.projectId}.changes.fragments.${fragmentName}`;
+        if (template) {
```

Comment:
Why do we need the `if` condition here as it will always evaluate as `true`?

Reply:
removed